src/components/navigation/header/config/HeaderNavigationConfig.tsx

```diff
@@ -95,17 +104,40 @@ export class HeaderNavigationConfig extends BaseConfig<HeaderNavigationConfigIns
     return this.mappedConfig;
   }
 
+  replacePathParams = (
+    link: LocalizedLinks | undefined
+  ): LocalizedLinks | undefined => {
+    if (link && this.urlPathParams) {
+      const transformedLink = Object.entries(link).reduce(
+        (acc, [key, value]) => {
+          return {
+            ...acc,
+            [key]: replaceParameters({
+              path: value,
+              params: this.urlPathParams,
+            }),
+          };
+        },
+        {} as LocalizedLinks
+      );
+
+      return transformedLink;
```

Comment:
you could return that directly since we are not doing anything with the variable